opentelemetry-api/src/opentelemetry/util/tracestate.py

```diff
@@ -1,73 +0,0 @@
-# Copyright The OpenTelemetry Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
```

Comment:
Why delete this? I would just rename this to `_tracestate.py`

Reply:
All the content from this file was being used in only one place, I moved it where it was being used.